python_build_requirements.txt

```diff
@@ -6,3 +6,4 @@ black>=21.10b0
 mypy>=0.910
 ni-python-styleguide~=0.1
 pycodestyle==2.7.0
+importlib-metadata==4.12.0
```

Comment:
Note: This is a somewhat unrelated change but the build started breaking today when running python styleguide check on the python files: https://github.com/ni/grpc-device/actions/runs/3175907824/jobs/5174555407

importlib-metadata released version 5.0.0 which removed some deprecated methods I think we were hitting in the styleguide check. @astarche , I'm not too familiar with this stuff. Does this seem like a reasonable way to resolve this?